spec/models/post_spec.rb

```diff
@@ -0,0 +1,54 @@
+require 'rails_helper'
+
+RSpec.describe Post, type: :model do
+  # ...
+
+  it 'Title attribute should be less than 250 characters' do
+    subject.title = 'Your title with less than 250 characters' # Provide a valid title
+    expect(subject).to be_valid
+  end
+
+  it 'Comments Counter attribute should be an integer number' do
+    subject.comments_counter = 'some random string'
+    expect(subject).to_not be_valid
+  end
+
+  it 'Comments Counter attribute should be greater or equal to zero' do
+    subject.comments_counter = -4
+    expect(subject).to_not be_valid
+  end
+
+  it 'Likes Counter attribute should be an integer number' do
+    subject.likes_counter = 'some random string'
+    expect(subject).to_not be_valid
+  end
+
+  it 'Likes Counter attribute should be greater or equal to zero' do
+    subject.likes_counter = -4
+    expect(subject).to_not be_valid
```

Comment:
- [ ] Great job working on the required changes but I think it would be great if you solve the failing tests here since you have some falling tests in post and comment and like and user models I will include the screenshots to be able to check the error messages 💯🔥✅💪🏻 so kindly try to solve them 💯 to be passed 👏🏻🚀


- 1
![image](https://github.com/AlyAtalla/blog-app/assets/58553711/1e0202e1-08f0-4e83-b2c9-6552603b3b81)

- 2
![image](https://github.com/AlyAtalla/blog-app/assets/58553711/d6eefab2-cc90-4053-afca-09bff6e6c49f)

- 3
![image](https://github.com/AlyAtalla/blog-app/assets/58553711/b5df9ca8-1b9e-4b23-b19d-7cc9252b392f)